packages/text-input/src/TextInput.tsx

```diff
@@ -0,0 +1,328 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { css, cx } from '@leafygreen-ui/emotion';
+import Icon from '@leafygreen-ui/icon';
+import { uiColors } from '@leafygreen-ui/palette';
+import { useUsingKeyboardContext } from '@leafygreen-ui/leafygreen-provider';
+import { createDataProp } from '@leafygreen-ui/lib';
+
+export const State = {
+  none: 'none',
```

Comment:
Quick note: typically we PascalCase enum values (this being a pseudo-enum).

```js
export const State = {
  None: 'none',
  ...
};
```

Reply:
Fixed